geode-core/src/main/java/org/apache/geode/internal/cache/versions/RegionVersionVector.java

```diff
@@ -144,22 +145,56 @@
   private final transient Object clearLockSync = new Object(); // sync for coordinating thread
                                                                // startup and lockOwner setting
 
-  /** create a live version vector for a region */
+  /**
+   * constructor used to create a cloned vector
+   */
+  protected RegionVersionVector(T ownerId, ConcurrentHashMap<T, RegionVersionHolder<T>> vector,
```

Comment:
vector is an odd parameter name for a ConcurrentHashMap. Since this is a copy constructor I'd expect "vector" to be the RegionVersionVector we are copying. So I suggest renaming this parameter and updating the comment since we are not passing in the "vector" we are cloning.

Reply:
I see that you just moved this guy. I think my comment still is valid but not really part of your PR